### PR TITLE
Fetch packages from package-storage

### PR DESCRIPTION
**Pull request by mtojek**
Changes:
* modify the `mage build` to include packages from package-storage
* allow to run `mage build` multiple times (don't re-checkout the package-storage)
* ~remove `dev/packages/alpha` as they are in the package-storage~ (import recent alpha packages)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,5 @@
 .idea
 build
 public
+
+/dev/packages/storage
```
Comment (ruflin) [inline .gitignore:7]: Should we put it under `build/packages/storage` instead?

Comment (mtojek) [reply .gitignore]: Yes, this one doesn't need special cleaning. Fixed.

File: magefile.go [hunk 2]
```diff
@@ -30,10 +30,12 @@ var (
 	// GoLicenserImportPath controls the import path used to install go-licenser.
 	GoLicenserImportPath = "github.com/elastic/go-licenser"
 
-	publicDir    = "./public"
-	buildDir     = "./build"
-	packagePaths = []string{"./dev/packages/alpha/", "./dev/packages/example/", "./dev/packages/beats/"}
-	tarGz        = true
+	publicDir      = "./public"
+	buildDir       = "./build"
+	storageDir     = "./dev/packages/storage"
```
Comment (ruflin) [inline magefile.go:35]: Is this still correct?

**Comment by exekias** [comment]
ey folks, trying to understand the workflow here. Why is integrations repo fetching packages from storage?

File: magefile.go (hunk 2 above)
Comment (mtojek) [reply magefile.go]: Leftover, not needed anymore.

**Comment by mtojek** [comment]
> ey folks, trying to understand the workflow here. Why is integrations repo fetching packages from storage?

Packages served by the package-registry are stored in the package-storage. There are some of them that are internal/required like: `base`, `endpoint`, `system`. As they're not in the integrations repo, we need to fetch them from the package-storage.

**Comment by ruflin** [comment]
One reason are the required packages but there is also a very nice side effect here. As all the existing packages are pulled in, any testing of new packages together with Kibana is what it will look like in the end. Example:

Someone is working on the nginx package 1.3.2. The current version shipped is 1.3.1. 1.3.1 will be pulled in from the package-storage and 1.3.2 will be on top of it.

**Comment by mtojek** [comment]
> * remove `dev/packages/alpha` as they are in the package-storage

This point depends on outcome of the discussion here: https://github.com/elastic/integrations/issues/15

**Pull request merged**